api/controllers/UserController.ts

```diff
@@ -0,0 +1,43 @@
+import * as Package from 'pjson';
+import { Controller, Get, BaseRequest, BaseResponse, Post } from 'ts-framework';
+import {User} from '../models';
+import { UserService, ExtractService } from '../services/';
+
+@Controller('/user')
+export default class UserController {
+
+  @Get('/:id')
+  static async getUser(req: BaseRequest, res: BaseResponse) {
+    return res.success({
+      user: await UserService.getUser(req.params.id)
+    });
+  }
+
+  @Get('/:id/kyc')
+  static async getKycStatusForUser(req: BaseRequest, res: BaseResponse) {
+
```

Comment:
When creating empty endpoints, prefer throwing an exception so your request won't get stuck in the express handler.

```typescript
@Get('/:id')
static async getKycStatusForUser(req: BaseRequest, res: BaseResponse) {
  throw new HttpError('Not implemented yet', HttpCode.Server.INTERNAL_SERVER_ERROR);
}
```